dev_guide/authentication.adoc

```diff
@@ -103,3 +114,20 @@ for user input to confirm (`y/N` kind of input) about connecting insecurely.
 
 CLI configuration files allow you to easily
 link:../cli_reference/manage_cli_profiles.html[manage multiple CLI profiles].
+
+// tag::systemadminlogin[]
+
+[NOTE]
+====
+If you have access to system administrator credentials but have logged out of
+the link:../architecture/core_concepts/projects_and_users.html#users[default
+system user] *system:admin*, you can log back in as this user with the following
+command as long as the credentials are still present in your *_~/.kube/config_*
```

Comment:
Ansible copies the generated admin.kubeconfig to /root/.kube/config and ~/.kube/config of the ansible_user if that user != root. However if those files already exist it won't overwrite them. So if you've partially removed an install and re-run ansible generating a new admin.kubeconfig those files will be stale and may not work.